playkit/src/main/java/com/kaltura/playkit/player/TrackSelectionHelper.java

```diff
@@ -124,7 +124,7 @@
 
         void onTracksOverrideABRError(PKError pkError);
         void onUnsupportedVideoTracksError(PKError pkError);
-
+        void onNoVideoAudioTrackAvailableError(PKError pkError);
```

Comment:
lets change to noTracksAvaialbleError and add the text tracks to the validation if no text tracks

Reply:
Done